test/compilable/cppmangle.d

```diff
@@ -44,9 +43,24 @@ void test1()
     c.bar(4, 5, 6);
 }
 
-static assert(foo.mangleof == "_Z3fooiii");
-static assert(foob.mangleof == "_Z4foobiii");
-static assert(C.bar.mangleof == "_ZN1C3barEiii");
+version (linux)
+{
+    static assert(foo.mangleof == "_Z3fooiii");
```

Comment:
We need a better version identifier here. GDC uses this style of C++ mangling for all the platforms, including Windows (G++ and MSVC++ runtimes are not compatible, and we'll only link to code built by the former anyway).